sentences/de/lock_HassTurnOff.yaml

```diff
@@ -0,0 +1,11 @@
+language: de
+intents:
+  HassTurnOff:
+    data:
+      - sentences:
+          - "<entsperren> <name> [<area>]"
+          - "(<sperren>|<machen>) <name> [<area>] auf"
+          - "<name> [<area>] (auf[<sperren>|machen]|<entsperren>)"
```

Comment:
I added "aufsperren" in the expansion rule but I think we need <sperren> in the second sentence in TurnOff, otherwise the sentence won't work as we have "auf" in the end here?